tests/api2/jail_plugins.py

```diff
@@ -0,0 +1,291 @@
+    assert plugin_repos in results.json(), results.text
+
+
+@to_skip
+def test_05_get_list_of_available_plugins_job_id():
+    global JOB_ID
+    payload = {
+        "plugin_repository": plugin_repos
+    }
+    results = POST('/plugin/available/', payload)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    JOB_ID = results.json()
+
+
+@to_skip
+def test_06_verify_list_of_available_plugins_job_id_is_successfull():
+    global job_results
+    while True:
+        job_results = GET(f'/core/get_jobs/?id={JOB_ID}')
+        job_state = job_results.json()[0]['state']
+        if job_state in ('RUNNING', 'WAITING'):
+            sleep(3)
+        else:
+            assert job_state == 'SUCCESS', str(job_results)
+            break
+
+
+@to_skip
+@pytest.mark.parametrize('plugin', plugins_list)
+def test_07_verify_available_plugin_(plugin):
+    for plugin_info in job_results.json()[0]['result']:
+        if plugin in plugin_info:
+            assert isinstance(plugin_info, list), job_results.text
+            assert plugin in plugin_info, job_results.text
+
+
+@to_skip
+def test_08_add_transmision_plugins():
+    global JOB_ID
+    payload = {
+        "plugin_name": "transmission",
+        "jail_name": "transmission",
+        'props': [
+            'nat=1',
+            'vnet=1',
+            'vnet_default_interface=auto'
+        ],
+        "plugin_repository": plugin_repos,
+    }
+    results = POST('/plugin/', payload)
+    assert results.status_code == 200, results.text
+    JOB_ID = results.json()
+
+
+@to_skip
+def test_09_verify_transmision_plugin_job_is_successfull():
+    while True:
+        job_status = GET(f'/core/get_jobs/?id={JOB_ID}').json()[0]
+        if job_status['state'] in ('RUNNING', 'WAITING'):
+            sleep(3)
+        else:
+            assert job_status['state'] == 'SUCCESS', str(job_status)
+            break
+
+
+@to_skip
+def test_10_search_plugin_transmission_id():
+    results = GET('/plugin/?id=transmission')
+    assert results.status_code == 200, results.text
+    assert len(results.json()) > 0, results.text
+
+
+@to_skip
+def test_11_get_transmission_plugin_info():
+    global transmission_plugin
+    results = GET('/plugin/id/transmission/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    transmission_plugin = results.json()
+
+
+@to_skip
+def test_12_get_transmission_jail_info():
+    global transmission_jail
+    results = GET("/jail/id/transmission")
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    transmission_jail = results.json()
+
+
+@to_skip
+@pytest.mark.parametrize('object', plugins_objects)
+def test_13_verify_transmission_plugin_value_with_jail_value_of_(object):
+    assert transmission_jail[object] == transmission_plugin[object], results.text
```

Comment:
`object` is a built in variable in python, i would recommend we use some other name here perhaps, like `prop` ? As that accurately reflects the value